examples/correlation/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+source ../../conf/env.sh
+
+# Data file is from Spark Examples (data/mllib/sample_linear_regression_data.txt) and put in examples/data
+# The data file should be copied to $HDFS_ROOT before running examples
+DATA_FILE=$HDFS_ROOT/data/sample_linear_regression_data.txt
```

Comment:
```suggestion
DATA_FILE=$HDFS_ROOT/data/sample_correlation_data.txt
```
could you create a sample data using the data in the example?